graphql/util.go

```diff
@@ -387,28 +382,11 @@ func generateBuildVariants(ctx context.Context, versionId string, buildVariantOp
 		result = append(result, &pbv)
 	}
 
-	timeToGroupTasks := time.Since(groupTasksStartTime)
-
-	sortTasksStartTime := time.Now()
 	// sort variants by name
 	sort.SliceStable(result, func(i, j int) bool {
 		return result[i].DisplayName < result[j].DisplayName
 	})
 
-	timeToSortTasks := time.Since(sortTasksStartTime)
-
-	totalTime := time.Since(start)
```

Comment:
@khelif96 can you confirm you don't need this anymore? (Otherwise I can leave it in and remove the ticket number, if you intent to make an alert on this condition. Otherwise I don't think we're really going to see it)

Reply:
Yeah feel free to remove it!